source/_components/etherrain.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "EtherRain"
+description: "The EtherRain networked irrigation controller"
+date: 2017-08-09 06:16
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: etherrain.jpg
+ha_category: Hub
+featured: false
+ha_release: 0.31
```

Comment:
HA release will most like need to be updated to 0.52  since 0.51 is rather imminent.